.github/workflows/publish_docs_to_wiki.yml

```
@@ -13,6 +13,16 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 30
     steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.8.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout the repository
```

I think we should start the name of each step with a verb 🤔 
```suggestion
      - name: Check out the repository
```

---

Updated in [7ba2728](https://github.com/nimblehq/android-templates/pull/456/commits/7ba27285d6bdfe75d563fb184228444848006a17)